pkg/api/cookiestore.go

```diff
@@ -110,6 +110,10 @@ func getExpiredSessions(dir string) ([]string, error) {
 		return nil
 	})
 
+	if os.IsNotExist(err) {
```

Comment:
I saw some errors about this one in our tests, if rootDir/_sessions is empty then generator should finished for the moment to not keep the workers busy.